### PR TITLE
Added dataset links on dataland.

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

File: templates/Dataland/datasetlinks.html.twig [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<h2>Dataset Links:</h2>
+<div>
+    {% for link in dataset.datasetSubmission.datasetLinks %}
+        <p>
+            <b>Name:</b>{{ link.name }}<br>
+            <b>Description:</b>{{ link.description }}<br>
+            <b>Function:</b>{{ link.functionCode }}<br>
```
Comment (fightingtexasaggie) [inline templates/Dataland/datasetlinks.html.twig]: we just showing the raw function code here, or turning it into it's textual representation?

File: templates/Dataland/datasetlinks.html.twig [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<h2>Dataset Links:</h2>
+<div>
+    {% for link in dataset.datasetSubmission.datasetLinks %}
+        <p>
+            <b>Name:</b>{{ link.name }}<br>
+            <b>Description:</b>{{ link.description }}<br>
+            <b>Function:</b>{{ link.functionCode }}<br>
+            <b>URL:</b>
+            <a href="{{ link.url }}" target="_blank">
```
Comment (fightingtexasaggie) [inline templates/Dataland/datasetlinks.html.twig]: Should we decompose the link and strip out any included protocol, then add the protocol + link?

File: templates/Dataland/datasetlinks.html.twig (hunk 1 above)
Comment (mickel1138) [reply templates/Dataland/datasetlinks.html.twig]: Was working on that, now fixed.

File: templates/Dataland/datasetlinks.html.twig (hunk 2 above)
Comment (mickel1138) [reply templates/Dataland/datasetlinks.html.twig]: No, the protocol of the link may not be the same as the protocol. It's just a thing.

File: templates/Dataland/datasetlinks.html.twig [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<h2>Dataset Links:</h2>
+<div>
+    {% set linkcodes = constant('App\\Entity\\DatasetLink::ONLINE_FUNCTION_CODES') %}
+    {% for link in dataset.datasetSubmission.datasetLinks %}
+        <p>
+            <b>Name:</b>{{ link.name }}<br>
```
Comment (praneethpr) [inline templates/Dataland/datasetlinks.html.twig]: Need to add space for all the elements

File: templates/Dataland/datasetlinks.html.twig [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<h2>Dataset Links:</h2>
+<div>
+    {% set linkcodes = constant('App\\Entity\\DatasetLink::ONLINE_FUNCTION_CODES') %}
+    {% for link in dataset.datasetSubmission.datasetLinks %}
+        <p>
+            <b>Name:</b>{{ link.name }}<br>
+            <b>Description:</b>{{ link.description }}<br>
+            <b>Function:</b>{{ attribute(linkcodes[link.functionCode], 'name') }}<br>
```
Comment (praneethpr) [inline templates/Dataland/datasetlinks.html.twig]: Should we add the same style as summary tab?

Comment (mickel1138) [reply templates/Dataland/datasetlinks.html.twig]: Which style do you suggest? I tried h2, didn't look right. And the new lines use too much space I think.

Comment (praneethpr) [reply templates/Dataland/datasetlinks.html.twig]: We can ask the PO because it is a public facing feature

**Pull request merged**